pkgs/development/libraries/libgnt/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, lib, fetchurl, meson, ninja, pkg-config, python2
+, gtk-doc, docbook-xsl-nons
+, glib, ncurses, libxml2
+}:
+let
+  # Pidgin will get build-time dependency on gtk-doc anyway (through gstreamer).
+  docs = true;
+in
+stdenv.mkDerivation rec {
+  pname = "libgnt";
+  version = "2.14.1";
+
+  outputs = [ "out" "dev" ] ++ lib.optional docs "devdoc";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/pidgin/${pname}-${version}.tar.xz";
+    sha256 = "1n2bxg0ignn53c08cp69pj4sdg53kwlqn23rincyjmpr327fdhsy";
+  };
+
+  nativeBuildInputs = [ meson ninja pkg-config ]
+    ++ lib.optionals docs [ gtk-doc docbook-xsl-nons ];
+
+  # python2 is optional but Pidgin does depend on it (and python3 won't be accepted in 2.x)
+  buildInputs = [ glib ncurses libxml2 python2 ];
+
+  postPatch = ''
+    substituteInPlace meson.build --replace \
+      "ncurses_sys_prefix = '/usr'" \
+      "ncurses_sys_prefix = '${lib.getDev ncurses}'"
+  ''
+  + lib.optionalString (!docs) ''
+    sed "/^subdir('doc')$/d" -i meson.build
+  '';
+
+  meta = with lib; {
+    description = "An ncurses toolkit for creating text-mode graphical user interfaces";
+    homepage = "https://keep.imfreedom.org/libgnt/libgnt/";
+    license = licenses.gpl2Plus;
+    platforms = platforms.unix;
```

Comment:
Missing maintainer.

Reply:
For this one I need maintainer. I hope maybe @vcunat can become maintainer of `libgnt`.

Reply:
Otherwise please add yourself because otherwise we cannot merge it.

Reply:
Ok. I'll use myself for now. Though not sure I'll have enough time to monitor it.  
If there is a way to mark "looking for maintainers" - let me know :slightly_smiling_face: